tasks/Module 3/task_1.py

```diff
@@ -0,0 +1,93 @@
+"""Module2/task1: Includes task of creating sandwich calculator based on user's input."""
+import pyinputplus as pyip
+
+
+# Vars
+BREAD_TYPES_AND_PRICES = {'wheat': 4, 'white': 2, 'sourdough': 3}
+PROTEIN_TYPES_AND_PRICES = {'chicken': 6, 'turkey': 7, 'ham': 7, 'tofu': 4.85}
+CHEESE_TYPES_AND_PRICES = {'cheddar': 5, 'Swiss': 9, 'mozzarella': 7.5}
+SAUCE_TYPES_AND_PRICES = {'mayo': 2.5, 'mustard': 2, 'lettuce': 1, 'tomato': 0.75}
+
+
+def select_bread() -> dict:
```

Comment:
It is optional, but if you want more support from type hints, you can specify types for dict's keys and values. Unfortunately* it is a bit complicated, because it requires usage of [typing module](https://docs.python.org/3/library/typing.html). Long story short you can do:
```python
import typing
...
def select_bread() -> typing.Dict[str, float]:
```
This way your IDE will have better possibilities to analyze your code. Also when you have a lot of exactly the same typing annotations and they are complicated, then it is convenient to extract them to some variable:

```python
import typing
...
IngredientDict = typing.Dict[str, float]
...
def select_bread() -> IngredientDict:
```

Also [namedtuple](https://docs.python.org/3/library/collections.html#collections.namedtuple) and [dataclasses](https://docs.python.org/3/library/dataclasses.html) are worth checking :)

* In the future (Python 3.9) we will just write `dict[str, float]`

Reply:
That's nice. TY!